src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -538,23 +538,36 @@ export class AppComponent implements OnInit, AfterViewChecked {
      */
     xmlLinkHandler = (cellRef: string, row: number, dataContext: JSON, colDef: any) => {
         const self = this;
-        let value = dataContext[colDef.field];
-        $(cellRef).children('.xmlLink').click(function(): void {
-            self.dataService.openLink(value, colDef.name, 'xml');
-        });
+        self.handleLink(cellRef, row, dataContext, colDef, 'xml');
```

Comment:
Minor: you can use `this` instead of `self` here since we are no longer using it inside an embedded function. Same for the jsonLinkHandler method